root/utility/sanitizedContext.js

```diff
@@ -20,13 +20,17 @@ function sanitizedContext(
   $c /*: CatalystContextT */,
 ) /*: SanitizedCatalystContextT */ {
   const user = $c.user;
+  const genre_list = $c.stash.genre_list;
   return {
     action: {
       name: $c.action.name,
     },
     req: {
       uri: $c.req.uri,
     },
+    stash: genre_list ? ({
+      genre_list: $c.stash.genre_list,
+    }) : {},
```

Comment:
`$c.stash.genre_list` is stored in a variable already so you can simplify that a bit to `genre_list ? {genre_list} : {}`

Reply:
Though eslint might complain about the `genre_list` assignment above too, should probably name it `genreList` and do `{genre_list: genreList}` there